src/Polish.cpp

```diff
@@ -295,7 +318,15 @@ vector<int> ConsensusQualities(AbstractIntegrator& ai)
             if (m.Start() > i) continue;
             // TODO (lhepler): this is dumb, but untestable mutations,
             //   aka insertions at ends, cause all sorts of weird issues
-            const double score = ai.LL(m) - LL;
+            double score;
+            try {
+                score = ai.LL(m) - LL;
+            } catch (const Exception::InvalidEvaluatorException& e) {
+                // If an Evaluator exception occured, report and skip! 
+                // We need to handle this!
+                std::cerr << "In Polish::ConsensusQualities(ai):" << e.what() << std::endl;
```

Comment:
I was about to comment along these lines.  ConsensusCore[2] has never had "true" logging mechanism.  ConsensusCore had a trivial diagnostic logging capability here that enabled it to be turned on/off:

https://github.com/PacificBiosciences/ConsensusCore/blob/master/src/C%2B%2B/Logging/Logging.cpp


I agree that we shouldn't print to cerr right now, we should talk about the logging plan first